pkgs/development/python-modules/mayim/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+
+, psycopg
+, aiosqlite
+, asyncmy
+
+# test
+, coverage
+, pytest
+, pytest-cov
+, pytest-asyncio
+, tox
+
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mayim";
+  version = "1.1.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "ahopkins";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-nb0E9kMEJUihaCp8RnqGh0nSyDQo50eL1C4K5lBPlPQ=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  passthru.optional-dependencies = {
+    postgres = [
+      psycopg
+    ] ++ psycopg.optional-dependencies.pool;
+    mysql = [
+      asyncmy
+    ];
+    sqlite = [
+      aiosqlite
+    ];
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    coverage
```

Comment:
```suggestion
```
We enable testing to make sure the package works, so please remove coverage measurements.

Reply:
Their pyproject.toml defines arguments for coverage for pytest. Is there a way to overwrite this?

Reply:
Usually we override it with substituteInPlace in preCheck.

Reply:
OK, I did patch that out as a workaround